ovs/bootstrap

```
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+build_dir="/build"
+ovs_ver="2.7.0"
+build_deps="build-essential libssl-dev python python-six curl"
+
+function setup_build() {
+    apt-get update && apt-get install -y ${build_deps}
```

If `apt-get update` fails, will it fail silently?